aiida/backends/tests/orm/comments.py

```diff
@@ -73,36 +73,88 @@ def test_comment_collection_delete(self):
         with self.assertRaises(exceptions.NotExistent):
             Comment.objects.get(id=comment_pk)
 
+    # pylint: disable=too-many-locals
     def test_comment_querybuilder(self):
         """Test querying for comments by joining on nodes in the QueryBuilder."""
+        user_one = self.user
+        user_two = orm.User(email="commenting@user.s").store()
+
         node_one = orm.Data().store()
-        comment_one = Comment(node_one, self.user, 'comment_one').store()
+        comment_one = Comment(node_one, user_one, 'comment_one').store()
 
         node_two = orm.Data().store()
-        comment_three = Comment(node_two, self.user, 'comment_three').store()
-        comment_four = Comment(node_two, self.user, 'comment_four').store()
+        comment_two = Comment(node_two, user_one, 'comment_two').store()
+        comment_three = Comment(node_two, user_one, 'comment_three').store()
+
+        node_three = orm.CalculationNode().store()
+        comment_four = Comment(node_three, user_two, 'new_user_comment').store()
+
+        node_four = orm.CalculationNode().store()
+        comment_five = Comment(node_four, user_one, 'user one comment').store()
+        comment_six = Comment(node_four, user_two, 'user two comment').store()
 
         # Retrieve a node by joining on a specific comment
-        nodes = orm.QueryBuilder().append(
-            Comment, tag='comment', filters={
-                'id': comment_one.id
-            }).append(
-                orm.Node, with_comment='comment', project=['uuid']).all()
+        # pylint: disable=invalid-name
```

Comment:
I would suggest instead of using the pylint exception to just rename `qb` to `builder`.

Reply:
Done - remove disabled pylint keys as well. Doing this going forward.